### PR TITLE
refactor(config): RHICOMPL-1894 Kafka config read from Settings

### PR DESCRIPTION
**Pull request by vkrizan**
Read Kafka configuration from `Settings` for ease of Clowder config addition.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [X] Input Validation
- [X] Output Encoding
- [X] Authentication and Password Management
- [X] Session Management
- [X] Access Control
- [X] Cryptographic Practices
- [X] Error Handling and Logging
- [X] Data Protection
- [X] Communication Security
- [X] System Configuration
- [X] Database Security
- [X] File Management
- [X] Memory Management
- [X] General Coding Practices

### CODE REVIEW

File: test/producers/application_producer_test.rb [hunk 1]
```diff
@@ -3,34 +3,36 @@
...
 
 class ApplicationProducerTest < ActiveSupport::TestCase
-  class MockProducer < ApplicationProducer
-    BROKERS = ['broker1'].freeze
+  setup do
+    @_kafka_settings = Settings.kafka
+  end
+
+  teardown do
+    Settings.kafka = @_kafka_settings
```
Comment (akofink) [inline test/producers/application_producer_test.rb:11]: Does `Settings.reload!` not do it?

**Review by akofink — APPROVED** [review]
Continues to work as expected. Thanks!

**Comment by skateman** [comment]
Looks good :+1:

File: test/producers/application_producer_test.rb (hunk 1 above)
Comment (vkrizan) [reply test/producers/application_producer_test.rb:11]: For some reason it did not. I can retry.

Comment (vkrizan) [reply test/producers/application_producer_test.rb]: It worked at the end. Thanks.

**Pull request merged**